working-groups/WORKING-GROUPS.md

```diff
@@ -201,22 +200,6 @@ interest include: load balancing, routing, DNS configuration and TLS support.
 | <img width="30px" src="https://github.com/tcnghia.png">   | Nghia Tran       | Google  | [tcnghia](https://github.com/tcnghia)     |
 | <img width="30px" src="https://github.com/mdemirhan.png"> | Mustafa Demirhan | Google  | [mdemirhan](https://github.com/mdemirhan) |
 
-## Observability
```

Comment:
I think it would be useful to archive the WG details in a separate file so the links to meeting notes, docs, and slack channel are preserved without having to dig through the team drive. We can also add an epitaph that explains when and why the WG was archived. This would be useful context for whoever tries to spin this WG up again (mentioned as a possibility in the TOC meeting today)

Reply:
I think Max is actually interested in a "performance" WG, which would have a different focus from "observability" in that they would be looking primarily at how to improve measurable system qualities, rather than providing developers insight when their applications on the platform are failing.

(You could send a PR to Matt's branch for this.)

Reply:
We didn't do this for Build, and I'm not super worried about recovering context (given commit history).

A graveyard is just another thing to maintain, so I'm a bit reluctant to take that on.